### PR TITLE
React native/spanish

### PR DESCRIPTION
**Pull request by IvanSotelo**
This pull request implements the spanish translation for the learnstorybook for Reac Native. Ref #3

### CODE REVIEW

**Comment by jonniebigodes** [comment]
@IvanSotelo wow...this is awesome, would you mind if i take a look at it tomorrow(wednesday) and come back with a review if needed? I'm in the middle of applying the finishing touches on another pr that will be up soon, one that you might need to be involved as you spearheaded the translation effort for spanish.  Do you mind waiting a bit?

**Comment by IvanSotelo** [comment]
Of course, no problem.

**Comment by jonniebigodes** [comment]
@IvanSotelo sorry for the delay, but i've gone through it and nothing stands out. Now your pr and a recent comment on the original pr that was made for the React Native led me to do some digging and it seems that there's some changes that will need to be addressed. So with that in mind we have options here:
1- We can merge this as is and make a subsequent pr when the original documentation is fixed.
2- We wait a bit, while i go through the original documentation and the latest Storybook and Expo/React Native approach, i make the changes, you adjust accordingly in the translation and in one fell swoop we make available the most accurate documentation. 

I was planning on revisiting some of the frameworks during the weekend so this presented as a golden opportunity

I would like to hear from you on how to proceed and also @domyen and @tmeasday your input aswell would be very much appreciated in this.

And also it goes without saying, me personally and i believe the team aswell thanks you for the time and effort you put into this

**Comment by jonniebigodes** [comment]
@IvanSotelo saw that you committed some changes, which is really awesome. I went through the tutorial during the weekend and it seems there's some things that need to be addressed. Nothing much, more precisely some code examples here and there. Now i have the changes in play locally. The reason i haven't pushed them up is because there's a issue with Jest, i've reached out and created a issue on the repo and also in their community forums. I'm going to "put a clock" (pardon the bad pun) on the issue. If no feedback is provided in either case till Wednesday, then Thursday i'm pushing the changes i have. You adjust accordingly and we get this merged as soon as possible so that you won't have this pr "blocked" for something that is unrelated to the effort and time you put in and allow the documentation to expand. Are you onboard with this?

**Comment by IvanSotelo** [comment]
@jonniebigodes i've adjusted the translation as we saw in your pr [#286](https://github.com/chromaui/learnstorybook.com/pull/286)

**Comment by jonniebigodes** [comment]
@IvanSotelo going to take a closer look at it a bit later. I've skimmed through it just now and is looking good so far.

File: content/intro-to-storybook/react-native/es/get-started.md [hunk 1]
```diff
@@ -0,0 +1,527 @@
+---
+title: 'Tutorial Storybook para React Native'
+tocTitle: 'Empezando'
+description: 'Configurar Storybook en tu entorno de desarrollo'
+---
+
+Storybook se ejecuta junto con tu aplicación en modo desarrollo. Te ayuda a crear componentes de interfaz de usuario aislados de la lógica y el contexto de tu aplicación. Esta edición de Aprende Storybook es para React Native; existe otras ediciones para [Vue](/vue/es/get-started), [Angular](/angular/es/get-started), [React](/react/es/get-started) y [Svelte](/svelte/es/get-started).
```
[the remark this answers is not in the document]
Comment (IvanSotelo) [reply content/intro-to-storybook/react-native/es/get-started.md]: @jonniebigodes  marking this as solved per latest commit

**Comment by jonniebigodes** [comment]
@IvanSotelo once again it goes without saying that this is amazing. Thank you for the time and effort you put into this. I've just ran through it and it's good to go. 

If you don't mind waiting a bit until the pr is merged,  it seems that the team is currently quite busy working on Storybook probably adding some more cool features so this could take a bit.

I saw that you expressed interest on translating the svelte documentation, which once again it's awesome. If you could start the initiative it would be awesome. I wrote the original version, should you need any help reviewing it let me know.

**Comment by IvanSotelo** [comment]
Ready @jonniebigodes , I have made some changes to review.

**Comment by jonniebigodes** [comment]
@IvanSotelo sorry for the delay. I've looked at the changes and it all checks out. i'm going to start by merging your pull request and a little later i'll merge the english version. 

As it never gets old, me personally and i believe the team aswell cannot thank you enough for the time and effort you've put into this.

**Pull request merged**